_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,190 @@
+---
+id: efbfc7
+name: Text content that updates automatically can be paused, stopped or hidden
+rule_type: atomic
+description: |
+  This rule checks that there are mechanisms to pause, stop or hide the auto-updating of text content.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgements:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [visible][] [HTML element][] in an [HTML document][] if:
+
+- the `innerText` property of the [element][html element] changes without the user [activating][activate] or [focusing][focus] any [element][html element] in the same [HTML document][];
+- the [element][html element] does not have [children][child] whose `innerText` property is also changed;
+- the change happens after the [readiness][document readiness] of the [HTML document][] the [element][html element] belongs to is equal to "complete";
+- it is not the only [content][] in the [HTML document][].
+
+## Expectation
+
+For the test target a [user interface component][] is provided to pause, stop or hide the change of the [text content][].
```

Comment:
Should that UI component be visible and in the accessibility tree and with a non empty name?
(will need corresponding failed examples)